Source/ThirdParty/ANGLE/src/libANGLE/renderer/metal/mtl_state_cache.h

```diff
@@ -319,15 +319,19 @@ struct RenderPassAttachmentDesc
     bool equalIgnoreLoadStoreOptions(const RenderPassAttachmentDesc &other) const;
     bool operator==(const RenderPassAttachmentDesc &other) const;
 
-    ANGLE_INLINE bool hasImplicitMSTexture() const { return implicitMSTexture.get(); }
+    ANGLE_INLINE bool hasResolveTexture() const { return resolveTexture.get(); }
 
+    // When rendering with implicit multisample, |texture| is the texture that
+    // will be rendered into and discarded at the end of a render pass. Its
+    // result will be automatically resolved into |resolveTexture|.
     TextureRef texture;
-    // Implicit multisample texture that will be rendered into and discarded at the end of
-    // a render pass. Its result will be resolved into normal texture above.
-    TextureRef implicitMSTexture;
+    TextureRef resolveTexture;
```

Comment:
With this change, `texture` is always the destination for rendering and `resolveTexture` is always the destination for the automatic resolve step.

I was going to change `texture` to `drawingTexture` but that added a lot of `renaming of member` to the patch, which I decided add too much noise.